examples/formik/js/async-validator/index.js

```diff
@@ -0,0 +1,84 @@
+// Helper styles for demo
+import './helper.css';
+
+import React from 'react';
+import { render } from 'react-dom';
+import { Formik } from 'formik';
+import { formValidation } from './validate';
+
+const App = () => (
+  <div className="app">
+    <h1>
+      Basic{' '}
+      <a
+        href="https://github.com/jaredpalmer/formik"
+        target="_blank"
+        rel="noopener"
+      >
+        Formik
+      </a>{' '}
```

Comment:
Remove `{' '}`

Reply:
Is just from formik examples, I want to keep them as similar as they are in their website... good news is that they have room from improvement :)